ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/ubuntu_deploy.py

```diff
@@ -92,6 +94,14 @@ def chat(bot):
         memory=user_form_data["Memory"],
     )
 
+    metadata = user_form_data.copy()
```

Comment:
instead of doing this, perhaps you can just do:
```
metadata['solution_name'] = user_form_data['Solution name']
metadata['solution_type'] = user_form_data....
```